lib/cache.js

```diff
@@ -45,30 +46,37 @@ class Cache {
     const tempFolder = path.join(os.tmpdir(), 'tldr', utils.uniqueId());
 
     // Downloading fresh copy
-    return Promise.all([
-      // Create new temporary folder
-      fs.ensureDir(tempFolder),
-      fs.ensureDir(this.cacheFolder)
-    ])
-      .then(() => {
-        // Download and extract cache data to temporary folder
-        return remote.download(tempFolder);
-      })
-      .then(() => {
-        // Copy data to cache folder
-        return fs.copy(tempFolder, this.cacheFolder);
-      })
-      .then(() => {
-        return Promise.all([
-          // Remove temporary folder
-          fs.remove(tempFolder),
-          index.rebuildPagesIndex()
-        ]);
-      })
-      // eslint-disable-next-line no-unused-vars
-      .then(([_, shortIndex]) => {
-        return shortIndex;
-      });
+    return (
+      Promise.all([
+        // Create new temporary folder
+        fs.ensureDir(tempFolder),
+        fs.ensureDir(this.cacheFolder),
+      ])
+        .then(() => {
+          // Download and extract cache data to temporary folder
+          let activeDownloads = [];
+          for (let lang of this.config.languages) {
+            activeDownloads.push(remote.download(tempFolder, lang));
+          }
+
+          return Promise.allSettled(activeDownloads);
```

Comment:
Could rewrite this to be a bit more succinct by using a map I think?
```
return Promise.allSettled(this.config.languages.map((lang) => remote.download(tempFolder, lang)));
```

Reply:
Looks great. I'll update the file.

Note: There is a linting rule "arrow-parens" problem with this, so I'll just include extra parentheses and update the code.